libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -93,6 +104,18 @@ + (instancetype)sharedManager
         } else {
             sdkManager = [[self alloc] init];
         }
+
+        /*
+         * Checks if an analytics app name has already been set by the app.
+         * If not, fetches the default app name to be used and sets it.
+         */
+        NSString *currentAiltnAppName = [SalesforceSDKManager ailtnAppName];
```

Comment:
Can we do this in `initialize`?  I'm generally not a fan of overloading the `sharedManager` method with this kind of stuff, since it's purpose is really to just deliver a singleton instance.  Static initialization is best done in `initialize`.